### PR TITLE
feat(frontend): Profile Page Responsiveness bb-513

### PR DESCRIPTION
**Pull request by KeidsID**
Related to issue #513, this PR will add mobile responsiveness for the `/profile` page.

https://github.com/user-attachments/assets/dc32654d-b915-4e95-ae76-001b6cce1d88

### CODE REVIEW

File: apps/frontend/src/pages/profile/profile.tsx [hunk 1]
```diff
@@ -86,18 +87,18 @@ const Profile: React.FC = () => {
...
 						<UpdateUserForm onSubmit={handleUpdateSubmit} user={user} />
-					</div>
-					<h4 className={styles["title-password"]}>Change your password</h4>
-					<div className={styles["password-container"]}>
+					</ProfileSection>
+					<ProfileSection title="Change Password">
 						<UpdatePasswordForm onSubmit={handleUpdatePasswordSubmit} />
-						<div className={styles["button-container"]}>
+					</ProfileSection>
+					<ProfileSection title="Danger Zone">
```
Comment (fshabanov) [inline apps/frontend/src/pages/profile/profile.tsx:97]: Not sure about using "Danger Zone" for a sign out. Discuss with QA

**Review by fshabanov — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (WilhelmWesser) [inline package-lock.json:1]: Please, pay attention to lock file. If there are no new dependencies it should not be changed

**Review by WilhelmWesser — CHANGES_REQUESTED** [review]

File: apps/frontend/src/pages/profile/profile.tsx [hunk 2]
```diff
@@ -86,18 +87,18 @@ const Profile: React.FC = () => {
 			{isLoading && <Loader />}
 			{user && (
 				<div className={styles["page-container"]}>
-					<h4 className={styles["title"]}>Profile</h4>
-					<div className={styles["content-container"]}>
+					<ProfileSection title="Profile">
```
Comment (marharita08) [inline apps/frontend/src/pages/profile/profile.tsx:90]: 'Profile' is title of the whole page, not of the section. I think it should have bigger font-size than other titles on this page.

**Comment by Mikoyzskie** [comment]
Also I made changes on some buttons on this page.

**Pull request merged**